src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.Cart/OrderHistory/Services/OrderService.ts

```diff
@@ -27,6 +27,11 @@ module Orckestra.Composer {
                 });
         }
 
+        public cancelOrder(orderNumber: string) {
+            this.orderRepository.cancelOrder(orderNumber).then(result => window.location.reload()) 
```

Comment:
please add new event to MyAccountEvents - OrderCanceled = 'orderCanceled' and publish it here with data 
{ OrderNumber }
also remove here .then(result => window.location.reload())